tests/robot-tests/tests/admin/analyst/role_ui_permissions/publication_owner_permissions.robot

```diff
@@ -29,7 +29,25 @@ Check cannot edit content for published release
 Navigate back to admin dashboard for publication
     user navigates to publication on admin dashboard    ${PUBLICATION_NAME}    ${THEME_NAME}    ${TOPIC_NAME}
 
+Check Edit publication page inputs are correct
+    ${accordion}=    user gets accordion section content element    ${PUBLICATION_NAME}
+    user clicks link    Manage this publication    ${accordion}
+    user waits until page contains title    Manage publication
+
+    user waits until page contains element    id:publicationForm-themeId
```

Comment:
I think we might have a way of checking for the existence of formfields by their labels in the Robot tests, which might be more readable.  Not 100% sure though, and def not high priority!